core/src/main/java/io/ap4k/kubernetes/decorator/AbstractAddProbeDecorator.java

```diff
@@ -30,15 +30,16 @@
 import java.util.Arrays;
 import java.util.Collections;
 
-public abstract class AbstractAddProbeDecorator extends Decorator<ContainerBuilder> {
+public abstract class AbstractAddProbeDecorator extends ApplicationContainerDecorator {
```

Comment:
Could we perhaps have a small comment on what the point of the class is?

Reply:
ApplicationContainerDecorator does have a comment.
Added a comment on AbstractAddProbeDecorator.